.circleci/config.yml

```diff
@@ -10,5 +10,5 @@ jobs:
       - checkout
       - run: yarn
       - run: yarn lint
-      - run: yarn size
+      - run: NODE_OPTIONS=--openssl-legacy-provider yarn size
```

Comment:
Added this to fix the following issue.
![image](https://user-images.githubusercontent.com/85158673/145816151-eae39455-cd13-496d-a4a8-2b6d24657cd1.png)
More Details here - https://github.com/webpack/webpack/issues/14532

Reply:
Thanks. I was close to switch to old good Travis.